telegram-bot/meetup.go

```diff
@@ -188,3 +197,8 @@ func reminder(ID int64, client mongo.Client, s1 *gocron.Scheduler) {
 		s1.Clear()
 	}
 }
+
+func sendErrorToChan(err error, ID int64, errorTxt string) {
+	log.Print(err)
+	bot.Send(tbot.NewMessage(ID, errorTxt))
+}
```

Comment:
declare this in `main.go` then it should be accessible across all the files. So you can call the function in other functions not part of meetups too.